try {
    throw "Boom !!!"
} catch (e) {
    console.error(e);
    console.error("Caught :)");
}

console.log("Hi there !")